pkg/fleetallocation/controller.go

```diff
@@ -279,7 +279,11 @@ func (c *Controller) allocate(f *stablev1alpha1.Fleet) (*stablev1alpha1.GameServ
 
 	gsCopy := allocation.DeepCopy()
 	gsCopy.Status.State = stablev1alpha1.Allocated
-
+    
+    if fam != nil {
```

Comment:
This will need formatting - if you `make shell` you can run: `goimports -w ./pkg/` and you will be good to go :+1: